app/src/main/assets/pdf/index.html

```diff
@@ -0,0 +1,29 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <meta charset="utf-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1">
+  <meta name="google" content="notranslate">
+  <title>PDF Viewer</title>
+
+  <style>
+    body {
+      background-color: #808080;
```

Comment:
I'm not sure how flexible PDF.js is with text color and background color? If it allows us to change colors we can inject them here to honor user's theme preference. But for a start we can go with black text on white background.

Reply:
Nah, this color is the background color of the document, and the pages go on top of it.
I.e.:

![image](https://user-images.githubusercontent.com/12795/33527762-e8c2b30a-d81b-11e7-96e2-803b834076f2.png)

It's somewhat similar to how Chrome renders PDFs:

![image](https://user-images.githubusercontent.com/12795/33527766-09441362-d81c-11e7-8a69-0d4f391780d8.png)

As for the actual document's text/background colors - it may be tricky, because a PDF doc may contain its own rules for the text/background colors. But since PDF.js just converts a PDF doc to a DOM tree with some CSS class names, it's probably possible to try to adjust them according to user's theme preference via custom CSS rules. Though since it's somewhat free-form, it again may be tricky to make it reliable and work consistently across different PDF documents..